<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>curveRigging</string>
  </property>
  <widget class="QCheckBox" name="if_rebuild_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>rebuildCurve</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="crv_points_num_sb">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="create_crv_ctrl_pb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>创建路径控制器</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cp_cb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CenterPivot</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="if_keep_shape_cb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>keep history</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>控制器权重：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="controls_weight_sb">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>199</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>157</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>控制器数量：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="controls_num_sb">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>162</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QSlider" name="ctrLweight_sl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>162</y>
     <width>113</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>最大权重值：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="controls_max_weight_sb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>282</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>曲线名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_crv_name_le">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="input_crv_name_pb">
      <property name="text">
       <string>《</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ctrLweight_sl</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>controls_num_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>controls_num_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ctrLweight_sl</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctrLweight_sl</sender>
   <signal>valueChanged(int)</signal>
   <receiver>controls_num_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
